network-api/networkapi/wagtailpages/factory/__init__.py

```diff
@@ -19,7 +19,9 @@
 
 
 def generate(seed):
+    # the first three entries are ordered the way they are to ensure correct nav ordering
```

Comment:
Glad you left this comment. 

Should we consider refactoring this at a later date so there's a _specific_ way to handle the nav ordering 9or nav creation)?

Reply:
the live nav gets built by looking at the show_in_menu status of a homepage.child, but since we're literally in the step before we have any of those, and given how infrequently we change the front page, I think it's probably fine to leave it as is rather than adding code to running an explicit ordering. As long as we keep the comments that explain why things do what they do =)